packages/portal-proto/src/pages/_app.tsx

```diff
@@ -1,5 +1,4 @@
 import tailwindConfig from "../../tailwind.config";
-import store from "../app/store";
```

Comment:
This `store` is not being used in the app. Hence I removed it along with the `Provider`.

Reply:
This is the only change in this file.